### DISCUSSION TITLE
iSCSI target/initiatior modes

### DISCUSSION THREAD

**Discussion by yottatsa** [BlueSCSI/BlueSCSI-v2#148] [Ideas]
BlueSCSIv2 w/ wi-fi can potentially terminate networked iSCSI target (remote volume or anything else) and expose it on the wire interface. Similarly, it can be iSCSI target and provide access to the SCSI device over the network.

Re: https://cloudisland.nz/@xssfox/112403751047475437 and https://xantronix.social/@xan/112411515186474612

**Comment by erichelgeson** [comment]
While conceptually this would be quite awesome, I'm not sure how practical it would be. The Pico's only has a 11mbps 2.4ghz WiFi module, and we rely on the TCP/IP stack of the host currently. While I believe this likely _could_ be implemented, I'm just not sure how useful it would be in practice with the limitations of the PicoW. 

This is not to discourage anyone from attempting this - please let us know how it goes if you do.
